src/custom/utils/operator.ts

```diff
@@ -0,0 +1,131 @@
+import { ChainId } from '@uniswap/sdk'
+import { OrderCreation, ORDER_KIND_SELL } from 'utils/signatures'
+
+/**
+ * See Swagger documentation:
+ *    https://protocol-rinkeby.dev.gnosisdev.com/api/
+ */
+const API_BASE_URL: Partial<Record<ChainId, string>> = {
```

Comment:
can you put these inside utils/api or sth? i will use them and therefore i think they belong decoupled from this :)

Reply:
it could be in the constants exported, at first i wanted to do it like that. But then I thought we could add in this file all the methods we use and consume from the API

I think it's not big enough to split, so kind of nice to leave this module self-contain imo